src/main/definitions/constants.ts

```diff
@@ -1,3 +1,13 @@
 export const URLS = {
   LEGACY_ET1: 'https://employmenttribunals.service.gov.uk/apply',
-} as const;
\ No newline at end of file
+} as const;
+
+export const CALLBACK_URL = '/oauth2/callback' as const;
```

Comment:
This one can live inside the existing URLs object and then be used like URLS.CALLBACK_URL. 

For the router links below it would be nice to group them together somehow. I've seen in the Divorce project these being called PageLinks.

Reply:
I didn't want to put them inside of the URLs object because we will be import this massive object when say we only need one link. But  if you think it will be better I can change it.

Reply:
It's okay for now but I'd like a consistent use of constants in future.